open-grant-proposals/open-proposal-kryptoseoulcommunity.md

```diff
@@ -0,0 +1,127 @@
+
+2. Collaboration on other existing hackathons where appropriate 
+- KryptoSeoul to see if there are existing hackathons in the schedule where Filecoin can join in, such as ETHVietnam, BUIDL series hackathons and conferences 
+
+3. Amplification of Filecoin Presence in Korea 
+- Help Filecoin on overall brand-building (not day to day operations) in Korea - look to KryptoSeoul to suggest activities here that may be suitable. Some potential activities:
+- Interviewing Juan Benet (Founder of Filecoin) on KryptoSeoul YouTube Channel (Erica Show) etc.
+- Explore 1-2 meetups in Korea for Filecoin, targeting developers (in Korean), conducted by Korean developer who understands the technicalities
+- Encourage developer communities to join our established channels for updates and regular engagement (Filecoin Slack and Telegram once they are set up)
+- Loop Filecoin in (where appropriate) into Blockchain or Crypto Summits or Conferences in Korea for speaking engagements 
+
+4. Introduction to Strategic Partners and Enterprises in Korea 
+Advise Filecoin on potential high leverage collaborations in Korea (and/or Southeast Asia), and make the introduction - such as Kakao (GroundX/Klaytn), LINE, Samsung, LG, universities; other blockchain projects
+
+## Value
+
+Filecoin is trying to grow in Korea in 2021. 
+KryptoSeoul will aid in the following goals:
+- Grow Filecoin presence in Korea, focusing on the overall brand and visbility
+- Engage and onboard more Developers and Filecoin enthusiasts in Korea through KryptoSeoul’s network
+- Build 1-2 strategic partnerships with large enterprises in Korea
+- Monitor Korean community feedback to advise any next steps required
+
+Potential risks of not getting it right:
+- Misinformation via miscommunication between Filecoin and KryptoSeoul, either due to language translations or others, leading to detriment to the brand
+
+Potential risks that will make executing on this project difficult: 
+- Sudden tightening of crypto landscape by Korean government (such as restricting activities)
+
+## Deliverables
+
+Listing out a few deliverables as below:
+- Successful promotion of asia hackathon season to Korea, direction of korean developers towards tribe hackathon in Singapore
+- Filecoin profiled on The Erica Show (either Juan or Colin)
+- Introduction to 2 potential strategic partners in Korea and help with relationship building 
+- At least 1 meetup planned in Korea this year for developers
+- 5 monthly recap (or report) of completed activities and feedback from Korean community by KryptoSeoul 
+
+## Development Roadmap
+
+Here is a tentative timeline for some of the activities, if there are additional events or conferences that Filecoin may benefit from, KryptoSeoul will add on:
+ 
+Aug ‘21
+- [community] Promotion of Filecoin Asia Hackathon Season and Filecoin official korean channels through community
+- [meet up] Preparation of developer 101 materials in Korean, plan small meetup for Sept ‘21 
+- Monthly report 
```

Comment:
Important for monthly report to include more than a list of activities for the month — we're looking for synthesis and insights as well.